src/components/Loading Screen/LoadingScreen.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components/macro'
+import BounceLoader from 'react-spinners/BounceLoader'
+
+export default function LoadingScreen() {
+  return (
+    <Wrapper>
+      <BounceLoader color={'var(--darker-grey)'} />
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
As an alternative, you could use a grid here
```suggestion
  display: grid;
  place-items: center;
```
https://1linelayouts.glitch.me/

If you have something similar multiple times in the app, think about a base class/component.

Reply:
At the moment not. This is my only component i have to super center. But if i have another i will add a component. Added a notice in my backlog to remind myself.